mobify/pinny
# Prevent open firing

Prevent the open function from running when pinny is already open. 

Status: Ready for review

Reviewers: @donnielrt @scalvert
## Changes
- Check for open class on pinny element before running open function
- Add event test to confirm only one open event is fired when pinny is opened twice
## How to test-drive this PR
- checkout this branch
- grunt test
- grunt serve 
- view pinny example page and open inspector
- attempt to open one of the example pinnys twice without closing it 
- opening animations should only execute once
